tests/parser/LgrTests.cpp

```diff
@@ -41,35 +41,36 @@ BOOST_AUTO_TEST_CASE(ReadLgrCollection) {
 RUNSPEC
 
 DIMENS
- 10 10 10 /
+  3 3 1 /
```

Comment:
We can switch the test back to the previous version, as the new features aren’t tested there. Or did I miss something?